src/app/@ansyn/ol/plugins/annotations/annotations-context-menu/components/annotation-label/annotation-label.component.less

```diff
@@ -1,11 +1,22 @@
 @import 'ansyn/less/colors';
+@import "ansyn/less/globals";
 
 :host {
+	font-family: @app-font;
```

Comment:
need to check the app and think for a better solution

Reply:
it was part of this https://github.com/AnSyn/imagery/commit/ae4e9e2bd6c428121d339239e5b6f926a0f331c8 commit. Let me know what do you want to do about it